src/Articles.jsx

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { ArticleList } from "./ArticleList";
+import Lottie from "lottie-react";
+import loading from "./assets/loading.json";
+import { PageSelection } from "./PageSelection";
+import { Filters } from "./Filters";
+
+export const Articles = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [page, setPage] = useState(1);
+  const [totalItems, setTotalItems] = useState(0);
+  const [filters, setFilters] = useState({
+    order: "desc",
+    sortBy: "created_at",
+    topic: "",
+  });
+  const limit = 4;
```

Comment:
This logic is outside the scope of the current task and should not be included yet